src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java

```diff
@@ -139,9 +143,11 @@ private void unrollAndRetry(Collection<SinkRecord> records) {
       try {
         writer.write(Collections.singletonList(record));
       } catch (TableAlterOrCreateException tace) {
+        log.debug(tace.toString());
         reporter.report(record, tace);
         writer.closeQuietly();
       } catch (SQLException sqle) {
+        log.debug(sqle.toString());
```

Comment:
Instead of putting debug here, put debug on sqlAllMessagesException in the next line as it does some sensitive log trimming